common/src/main/java/tachyon/conf/TachyonConf.java

```diff
@@ -404,4 +407,17 @@ private String lookupRecursively(final String base, Map<String, String> found) {
     }
     return resolved;
   }
+
+  /**
+   * Constants.USER_FILE_BUFFER_BYTES shuold not bigger than Integer.MAX_VALUE bytes.
+   * @throws IllegalArgumentException if USER_FILE_BUFFER_BYTES bigger than Integer.MAX_VALUE
+   */
+  private void checkUserFileBufferBytes() {
+    if (!this.containsKey(Constants.USER_FILE_BUFFER_BYTES)) { //load from hadoop conf
```

Comment:
NIT: we don't usually use `this` in Tachyon codebase by Tachyon convention. you can directly do `containsKey`